sdk/identity/identity/test/node/msalNodeTestSetup.ts

```diff
@@ -0,0 +1,229 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as dotenv from "dotenv";
+import Sinon, { createSandbox } from "sinon";
+import { MsalBaseUtilities } from "../../src/msal/utils";
+import { Recorder } from "@azure-tools/test-recorder";
+import {
+  AuthenticationResult,
+  ConfidentialClientApplication,
+  PublicClientApplication,
+} from "@azure/msal-node";
+import { PlaybackTenantId } from "../msalTestUtils";
+import { Test } from "mocha";
+
+dotenv.config({ path: ".env" });
```

Comment:
I _think_ the test recorder handles this for you now, but you should test it before listening to me :)

Reply:
Thanks for catching this! The recorder imports `env`, so it indirectly calls into `dotenv`.